tests/poison_detection/test_activation_defence.py

```diff
@@ -26,7 +26,7 @@
 from art.utils import load_mnist
 from art.visualization import convert_to_rgb
 
-from tests.utils import master_seed
+from tests.utils import master_seed, TestBase
```

Comment:
It looks like `TestBase` is not used.
```suggestion
from tests.utils import master_seed
```

Reply:
yes, you are right. It was left here unintentionally since I initially wanted to inherit TestBase for the test classes here in order to use overwritten assertEqual to compare numpy arrarys and then got more errors. Following the aforementioned change, I would also remove this.